libs/token/src/nft-token.service.ts

```diff
@@ -34,4 +37,17 @@ export class NFTTokenService extends TypeOrmCrudService<NFTToken> {
     req.options.query.join.contract = { eager: true };
     return super.getMany(req);
   }
+
+  async getTokenEvents(id: string): Promise<AssetsNftEvent[]> {
+    const nftToken = await this.nftTokenRepository.findOne(id);
+
+    if (!nftToken) {
+      throw new NotFoundException(`NFT with id ${id} not found`);
+    }
+
+    return this.nearIndexerService.findNFTEvents(
+      nftToken.contractId,
+      nftToken.tokenId || '0',
```

Comment:
what's the magic about '0' char ? )

Reply:
fixed

Reply:
I have noticed that some nft ids are stored in DB as null but they are equal to 0 in blockchain, so if there is no id it looks for '0'

Reply:
okay. Let’s keep it in mind though